src/main/java/com/denizenscript/depenizen/bukkit/properties/essentials/EssentialsPlayerProperties.java

```diff
@@ -50,7 +50,7 @@ public static EssentialsPlayerProperties getFrom(ObjectTag object) {
     };
 
     public static final String[] handledMechs = new String[] {
-            "is_afk", "god_mode", "is_muted", "socialspy", "vanish", "essentials_ignore"
+            "is_afk", "god_mode", "is_muted", "socialspy", "vanish", "essentials_ignore", "remove_essentials_home"
```

Comment:
This is very old legacy code, you don't have to fix it in this PR but it'd be helpful to update to modern format per https://github.com/DenizenScript/Denizen/issues/2355